db_model/orm_model.py

```diff
@@ -116,6 +116,7 @@ class BuildAction(Base):
     run_id = Column(Integer,
                     ForeignKey('runs.id', deferrable=True, initially="DEFERRED",
                                ondelete='CASCADE'))
+    # hash of the build command
```

Comment:
What is the problem when the name of the variable and the name of the column differs temporarily?